infrastructure/src/main/java/org/corfudb/infrastructure/orchestrator/actions/RestoreRedundancyMergeSegments.java

```diff
@@ -195,6 +206,30 @@ Layout restoreWithBackOff(CorfuRuntime runtime, StateTransferManager transferMan
                 .collect(Collectors.toList());
     }
 
+    /**
+     * Send the last transferred address as the committed tail to the target log unit.
+     * This is required to prevent loss of committed tail after state transfer finishes
+     * and then all other log units failed and auto commit service is paused.
+     */
+    private void transferCommittedTail(CorfuRuntime runtime, Layout layout, long committedTail) {
+        final int maxRetry = 3;
```

Comment:
please transform this variable into a filed (in the future probably we would like to make this parameter configurable via the app config)

Reply:
This is quite internal and not worth exposing to user.